src/main/java/domain/Dealer.java

```diff
@@ -0,0 +1,19 @@
+package domain;
+
+public class Dealer extends User {
+
+	private static final int UNDER_OVER_SCORE = 17;
+
+	public Dealer(final String name) {
+		this.name = name;
+	}
```

Comment:
```suggestion
	public Dealer() {
		super("딜러");
	}
```
상속할때 super 키워드를 보통 사용합니다
딜러는 굳이 파라미터로 이름을 받지 않고 내부적으로 "딜러"라는 이름을 지정해놓으면 따로 신경쓰지 않아도 될것 같네요

Reply:
오 사용할 때 훨씬 보기 좋은 것 같아요! 감사합니다